src/components/Garden/Feed/ProposalRankings.js

```diff
@@ -87,7 +87,7 @@ function Item({ icon, index, label, onSelect, selected }) {
           margin-left: ${0.75 * GU}px;
         `}
       >
-        {label}
+        {label === 'top' ? 'Most supported' : 'Newest'}
```

Comment:
Could we add this as a `label` attribute in the `iconsMapping` above and pass it down as prop?

So the rendered Item would look like: 


```
 <Item
          key={index}
          icon={getRankingIcon(item, selected === index)}
          index={index}
          label={getRankingLabel(item)}
          onSelect={onChange}
          selected={selected === index}
        />
```

where

```
function getRankingLabel(key) {
  return iconsMapping[key]['label']
}